spyder/widgets/findreplace.py

```diff
@@ -209,18 +209,31 @@ def eventFilter(self, widget, event):
     def create_shortcuts(self, parent):
         """Create shortcuts for this widget"""
         # Configurable
-        findnext = config_shortcut(self.find_next, context='find_replace',
-                                   name='Find next', parent=parent)
-        findprev = config_shortcut(self.find_previous, context='find_replace',
-                                   name='Find previous', parent=parent)
-        togglefind = config_shortcut(self.show, context='find_replace',
-                                     name='Find text', parent=parent)
-        togglereplace = config_shortcut(self.show_replace,
-                                        context='find_replace',
-                                        name='Replace text',
-                                        parent=parent)
-        hide = config_shortcut(self.hide, context='find_replace',
-                               name='hide find and replace', parent=self)
+        findnext = CONF.config_shortcut(
+            self.find_next,
+            context='find_replace',
+            name='Find next',
+            parent=parent)
+        findprev = CONF.config_shortcut(

```

Comment:
Please add a blank line above and below these calls to `CONF.config_shortcut`, as you did it for most other widgets in this PR.